srcpkgs/python3-ldap3/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'python3-ldap3'
+pkgname=python3-ldap3
+version=2.6.1
+revision=3
+wrksrc="ldap3-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-pyasn1"
+short_desc="Strictly RFC 4510 conforming LDAP V3 pure Python client (Python3)"
```

Comment:
"pure Python3 client" ? Or not even mention Python3, since it's in the package name and should be the default now.